multi-arch-builders/provisioning/ppc64le/variables.tf

```diff
@@ -0,0 +1,78 @@
+
+variable "ibmcloud_api_key" {
+    description = "Denotes the IBM Cloud API key to use"
+    default = ""
+}
+
+variable "ibmcloud_region" {
+    description = "Denotes which IBM Cloud region to connect to"
+    default     = "us-south"
+}
+
+#INSERTED FOR MULTI-ZONE REGION SUCH AS FRANKFURT
+
+variable "ibmcloud_zone" {
+    description = "Denotes which IBM Cloud zone to connect to - .i.e: eu-de-1 eu-de-2  us-south etc."
+    default     = "us-south"
+}
+
```

Comment:
interesting.. so some regions have only one zone and the name of the zone matches the region? I feel like later if they ever add a new zone to those regions it won't be good, but I guess that's their problem :)